### PR TITLE
feat: add support for SuiteCloud CLI

### PR DESCRIPTION
**Pull request by felixdivall**
Used for NetSuite development with SDF

### CODE REVIEW

**Comment by fedeci** [comment]
@felixdivall can you please sign the CLA?

**Comment by felixdivall** [comment]
Yes. Will do either later or tomorrow


Vänliga hälsningar/ Best regards

FELIX DIVALL

***@***.******@***.***>

+46 735 04 15 62





***@***.***

www.suitespot.se<http://www.suitespot.se/>

<https://aka.ms/o0ukef>
________________________________

File: src/suitecloud.ts [hunk 1]
```diff
@@ -0,0 +1,452 @@
...
+        "Sets up an account to use with the SuiteCloud CLI for Node.js",
+      options: [
+        {
+          name: ["-i", "--interactive"],
+          description: "Runs the command in interactive mode",
+        },
+        {
+          name: ["-d", "--dev"],
+          description:
+            "Specifies the NetSuite domain of the account you want to use. You only need to specify the NetSuite domain if you are not using a production account",
```
Comment (fedeci) [inline src/suitecloud.ts:94]: Is this description correct?

**Review by fedeci — CHANGES_REQUESTED** [review]
Thanks @felixdivall! I found a lot of missing args and I added them as suggestions, you can just review my suggestions and commit them. Also I left a couple notes on which I would like to get some feedback before merging this PR!

**Comment by felixdivall** [comment]
I have read the CLA Document and I hereby sign the CLA

File: src/suitecloud.ts (hunk 1 above)
Comment (felixdivall) [reply src/suitecloud.ts:110]: Yes. But I see how it can be confusing. 
SCR-20230426-w8g

**Comment by felixdivall** [comment]
Thanks for the feedback! I got Fig today, so haven't played around too much with it yet. Great get this response and fix my errors. I've commited most of your suggestions.

File: src/suitecloud.ts [hunk 2]
```diff
@@ -0,0 +1,452 @@
...
+        },
+      ],
+    },
+    {
+      icon: "🎫",
+      name: "account:savetoken",
+      description: "Saves a TBA token that you issued previously in NetSuite",
+      options: [
+        {
+          name: "--account",
```
[the remark this answers is not in the document]
Comment (fedeci) [reply src/suitecloud.ts:69]: Ping :)

**Comment by fedeci** [comment]
I missed some comma or some newline so prettier is complaining, can you please run `npm run lint:fix` and push the changes?

**Review by fedeci — APPROVED** [review]

**Comment by fedeci** [comment]
Thanks!

**Comment by felixdivall** [comment]
When will this be merged?

**Comment by fedeci** [comment]
Now, I was waiting for the CI and forgot to push this, sorry!

**Pull request merged**